dotCMS/src/integration-test/java/com/dotmarketing/portlets/containers/business/FileAssetContainerUtilTest.java

```diff
@@ -116,6 +135,70 @@ public void test_getHost_demo_default_demo() throws Exception {
+        final List<FileAsset> assets      = Arrays.asList(
+                this.createFileAsset("container", ".vtl",
+                        "$dotJSON.put(\"title\", \"Test VTL and Valid and Invalid Content Types\")\n" +
+                        "$dotJSON.put(\"max_contentlets\", 25)\n" +
+                        "$dotJSON.put(\"useDefaultLayout\",\"*\")", containerFolder, fileAssetContentType),
+                this.createFileAsset("default_container", ".vtl", "$title", containerFolder, fileAssetContentType));
+        final boolean showLive            = false;
+        final boolean includeHostOnPath   = false;
+
+        final Container container = FileAssetContainerUtil.getInstance().fromAssets(currentDefaultHost,
+                containerFolder, assets, showLive, includeHostOnPath);
+
+        Assert.assertNotNull(container);
+        Assert.assertTrue(container instanceof FileAssetContainer);
+        Assert.assertEquals(1   , FileAssetContainer.class.cast(container).getContainerStructuresAssets().size());
+        Assert.assertNotNull(FileAssetContainer.class.cast(container).getDefaultContainerLayoutAsset());
+        Assert.assertEquals("default_container.vtl",FileAssetContainer.class.cast(container).getDefaultContainerLayoutAsset().getFileName());
+    }
+
+    private FileAsset createFileAsset (final String fileName1,
+                                       final String fileExtension,
+                                       final String fileContent,
+                                       final Folder root1,
+                                       final ContentType fileAssetContentType) throws Exception {
+
+        final FileAsset fileAsset = new FileAsset();
+        final File tempFile1      = File.createTempFile(fileName1, fileExtension);
+        FileUtil.write(tempFile1, fileContent);
+        final String fileNameField1 = fileName1 + fileExtension;
+
+        fileAsset.setContentType(fileAssetContentType);
+        fileAsset.setFolder(root1.getInode());
+        fileAsset.setBinary(FileAssetAPI.BINARY_FIELD, tempFile1);
+        fileAsset.setStringProperty(FileAssetAPI.HOST_FOLDER_FIELD, root1.getInode());
+        fileAsset.setStringProperty(FileAssetAPI.TITLE_FIELD, fileNameField1);
+        fileAsset.setStringProperty(FileAssetAPI.FILE_NAME_FIELD, fileNameField1);
+        fileAsset.setIndexPolicy(IndexPolicy.FORCE);
+
+        // Create a piece of content for the default host
+        return APILocator.getFileAssetAPI().fromContentlet(
+                APILocator.getContentletAPI().checkin(fileAsset, user, false));
+    }
+
+    private Folder createAppContainerTest(final Host currentDefaultHost) throws DotDataException, DotSecurityException {
 
+        final String pathRoot   = "/application/containers/test"+System.currentTimeMillis();
+        final Folder rootFolder = APILocator.getFolderAPI().createFolders(pathRoot, currentDefaultHost, APILocator.systemUser(), false);
+        return rootFolder;
```

Comment:
Codacy found an issue: [Consider simply returning the value vs storing it in local variable 'rootFolder'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6347242)